scripts/partition_manager/partition_manager.rst

```diff
@@ -96,34 +99,83 @@ placement: dict
          Place the partition after the first existing partition in the list.
 
      Valid values in the lists are ``app``, ``start``, ``end``, or the name of any partition.
+     The value ``start`` refers to the flash device's start address.
+     The value ``end`` refers to its end address.
      It is not possible to place the partition after ``end`` or before ``start``.
 
       align: dict
          Ensure alignment of start or end of partition by specifying a dict with a ``start`` or ``end`` key respectively, where the value is the number of bytes to align to.
          If necessary, empty partitions are inserted in front of or behind the partition to ensure that the alignment is correct.
          Only one key can be specified.
-         Partitions which directly or indirectly (through spans) share size with the ``app`` partitions can only be aligned if they are placed directly after the ``app`` partition.
+         Partitions which directly or indirectly (through :ref:`spans <partition_manager_spans>`) share size with the ``app`` partitions can only be aligned if they are placed directly after the ``app`` partition.
+
+   An empty value is also permitted, like so:
```

Comment:
PM does not support empty placement values. Instead it requires all partition to have a placement defined relative to some other partition. I will create a PR which adds a check for this, causing a runtime error if it finds an empty 'placement'.

Reply:
https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1606

Reply:
Thanks for that! Did you see my comment above?
https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1605#discussion_r352927950

Any suggestions on how to fix the existing incorrect examples? `placement: {after: start}` or so?

Reply:
I didn't catch the meaning of your comment. 

You proposal would resolve the issue.